libs/application-generic/src/usecases/create-subscriber/create-subscriber.usecase.ts

```diff
@@ -20,10 +20,10 @@ import {
   UpdateSubscriber,
   UpdateSubscriberCommand,
 } from '../update-subscriber';
-import { CreateSubscriberCommand } from './create-subscriber.command';
+import { CreateOrUpdateSubscriberCommand } from './create-or-update-subscriber.command';
 
 @Injectable()
-export class CreateSubscriber {
+export class CreateAndUpdateSubscriberUseCase {
```

Comment:
thought: maybe we should use the term upsert here instead. `UpsertSubscriberUseCase`
```suggestion
export class CreateOrUpdateSubscriberUseCase {
```

Reply:
implemented the change

Reply:
resolving